src/bundle/Resources/public/js/OnlineEditor/buttons/ez-btn-attributes-edit.js

```diff
@@ -18,6 +18,10 @@ export default class EzBtnAttributesEdit extends EzWidgetButton {
             td: this.setDefaultClassesOnTableCells,
             li: this.setDefaultClassesOnListItems,
         };
+
+        this.nativeAttributes = {
+            table: ['border']
```

Comment:
I don't like to have this hardcoded here. 
I think it should be in the `eZ.richText` config.